sample-input/benchmarks/c5g7/c5g7-3d-rodded-B.py

```diff
@@ -0,0 +1,526 @@
+   [assembly_rfl_rod      , assembly_rfl_rod      , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_rod      , assembly_mox_rod      , assembly_rfl_unrod_rgt],
+   [assembly_mox_rod      , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_rod      , assembly_mox_rod      , assembly_rfl_unrod_rgt],
+   [assembly_mox_rod      , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_rod      , assembly_mox_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_mox_unrod    , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_rod      , assembly_mox_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_mox_unrod    , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_unrod    , assembly_mox_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_mox_unrod    , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]],
+  [[assembly_uo2_unrod    , assembly_mox_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_mox_unrod    , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
+   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]]])
+
+
+# Fill root cell with lattice
+root_cell.setFill(lattices[-1])
+
+
+###############################################################################
+##########################     Creating Cmfd mesh    ##########################
+###############################################################################
+
+openmoc.log.py_printf('NORMAL', 'Creating Cmfd mesh...')
+
+cmfd = openmoc.Cmfd()
+cmfd.setMOCRelaxationFactor(0.6)
+cmfd.setSORRelaxationFactor(1.0)
+cmfd.setLatticeStructure(51,51)
+cmfd.setOpticallyThick(True)
+cmfd.setGroupStructure([1,4,8])
+
+
+###############################################################################
+##########################   Creating the Geometry   ##########################
+###############################################################################
+
+openmoc.log.py_printf('NORMAL', 'Creating geometry...')
+
+geometry = openmoc.Geometry()
+geometry.setRootUniverse(root_universe)
+geometry.setZLevel(-20.0)
+geometry.setCmfd(cmfd)
+geometry.initializeFlatSourceRegions()
+
+###############################################################################
+########################   Creating the TrackGenerator   ######################
+###############################################################################
+
+openmoc.log.py_printf('NORMAL', 'Initializing the track generator...')
+
+track_generator = openmoc.TrackGenerator(geometry, num_azim, azim_spacing)
```

Comment:
If you decide to abstract the 3D geometry construction to "geometry.py" as some new "geometry3d" variable, then perhaps rename this file and keep these last 100 lines or so as a "c5g7-3d-slice.py" or something?